src/ui/tools/Theme.js

```diff
@@ -211,6 +211,12 @@ ui.defaultTheme = {
 		label: "m-input-label",
 		self: "m-input-self",
 	},
+	mapbox: {
+		body: "m-mapbox-body",
+		currentPoint: "m-mapbox-map-current-position-point",
```

Comment:
Rename to **"m-mapbox-current-point"**, it seems a bit too long of a name?

Reply:
Done!